CMSgov/cms-carts-seds
# MDCT-372: Get Help Page

# Description

Fixes [MDCT-372](https://qmacbis.atlassian.net/browse/MDCT-372)

![Get-Help-Attempt2](https://user-images.githubusercontent.com/19439679/174358673-43061077-5218-47a6-aa3a-e38e7279b1a2.png)


## How to test

`./dev local`
Click on Get Help link in the top right of the header or the contact us in the footer to check out the page!

## Dependencies

Please spell out any dependencies for this change -- ex -- requires an install / update / migration etc

# Get it done

Now that the PR is open this is what happens next

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
